POMs/homePage.ts

```diff
@@ -0,0 +1,82 @@
+import { expect, Locator, Page } from '@playwright/test';
+
+export class HomePage {
+    readonly page: Page;
+    readonly acceptCookiesButton: Locator;
+    readonly acccountLink: Locator;
+    readonly changeLanguageButton: Locator;
+    readonly locationModal: Locator;
+    readonly languageDropdown: Locator;
+    readonly countryDropdownn: Locator;
+    readonly confirmButton: Locator;
+    readonly heartButton: Locator;
+    readonly whishlistModal: Locator;
+    readonly addedToWishlistPopup: Locator;
+    readonly removedFromTheWishlist: Locator;
+    readonly wishlistButton: Locator;
+    readonly accessoriesLink: Locator;
+    readonly whishlistItem: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.acceptCookiesButton = page.getByRole('button', { name: 'Accept All Cookies' });
```

Comment:
this.acceptCookiesButton = page.locator("[id='onetrust-accept-btn-handler']");
it should work for every language :D